test/image-blueprints/group1/rhel93.toml

```diff
@@ -0,0 +1,19 @@
+name = "rhel-9.3"
+description = "Basic RHEL 9.2 edge image without MicroShift."
```

Comment:
```suggestion
description = "Basic RHEL 9.3 edge image without MicroShift."
```

Reply:
I was editing and testing on 3 different hosts and lost track of these :laughing: Thx